doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2996,36 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
+    
+.. _disable-a-user:
+
+Disable a User
+~~~~~~~~~~~~~~
+
+Disables an ``AuthenticatedUser`` whose ``identifier`` (without the ``@`` sign) is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/$identifier
```

Comment:
A few things:

- I'm planning on putting this outside "admin" so that it can be used remotely (instead of localhost only). Let me know if I shouldn't.
- We'll make this a full curl command, eventually. No rush.
- My version ends with `/$identifier/disable` to match other commands under the "users" API.

Reply:
**thought**: Please leave inside admin. Using the unblock key for outside localhost seems fine. When using this call has so many consequences, it should be admin-scoped.

Reply:
@poikilotherm It's worth a discussion for sure. 

From my superuser/administrator perspective on the Harvard Dataverse Repository, I'd like to be able to delete/disable a user instead of asking one of the people with access to the box to do so, but I also realize that I'm not super technical and there's the potential to mess something up with some of the /admin endpoints, so I understand that opening those up to me may not be the best option. 🤔 

If a user deletion request comes in, I see the natural escalation being that I try to delete the user, and if I can't do so, I disable them. I guess as long as they are in the same space (inside admin for example) I'd be fine with it. We don't see a lot of these requests, so it's not a huge issue in terms of support cycles, so if we make the decision to stay inside admin for now we can revisit later.

Reply:
We'll keep disable inside admin, so what the docs are reflecting is accurate!